docs/internals/howto-release-django.txt

```diff
@@ -52,6 +52,17 @@ Prerequisites
 
 You'll need a few things before getting started:
 
+* A Unix environment with the following tools installed (in alphabetical order):
+
+  * bash
+  * git
+  * GPG
+  * make
+  * man
+  * hashing tools (``md5sum``, ``sha1sum``, ``sha256sum``)
```

Comment:
The CLI tools on macOS are `md5` and `shasum` (the latter taking a `-a` option).

Perhaps... 

```suggestion
  * hashing tools (Typically ``md5sum``, ``sha1sum``, and ``sha256sum`` on Linux, or ``md5`` and ``shasum`` on macOS)
```